1048/en.md

```diff
@@ -0,0 +1,146 @@
+## Problem Restatement. 
+
+We Have an Array ```a``` of ```n+1``` elements, and in that we want to mark ```k``` points. And the output of this problem is the function
+
+```
+f(a,v) = max(a[i + 1] - a[i]) over all i in set of marked indices.
+```
+
+Our objective is to minimize ```f​```. 
```

Comment:
I don't think this is quite the problem similar to 1048. Your function finds the maximum difference between two consecutive numbers. You may simply remove this.